src/main/resources/assets/minestuck/models/item/double_ended_trident.json

```diff
@@ -0,0 +1,31 @@
+{
+    "parent": "item/handheld",
+    "textures": {
+        "layer0": "minestuck:item/double_ended_trident"
+    },
+    "display": {
+        "thirdperson_righthand": {
+            "rotation": [-44, -90, 0],
+            "translation": [0, -2, 0.55],
+            "scale": [1.25, 1.25, 1]
+        },
+        "thirdperson_lefthand": {
+            "rotation": [0, 90, -55],
+            "translation": [0, 5.5, 0.3],
+            "scale": [1.25, 1.25, 1.25]
+        },
```

Comment:
How its held in the third person left hand does not reflect how its held in third person right hand